pyproject.toml

```diff
@@ -19,6 +19,7 @@ license = { file="LICENSE" }
 dependencies = [
   'astpretty ~= 3.0',
   'cuquantum-cu11 ~= 23.10',
+  'graphlib-backport >= 1.0',
```

Comment:
What exactly is using graphlib?

Reply:
There is a topological sort needed for the kernel's dependent kernel call graph. I didn't realize graphlib was a 3.9+ library.

Reply:
Ah, I see. Not urgent to address, but I think we should consider just writing the topological sort ourselves and getting rid of this dependency. 
Edit: I just checked eol support; I would say we should drop support for Python versions when the security support ends, which would be October 24 for Python 3.8. If we don't have any issue with the backport library, we can leave it as is until then.